src/index.css

```diff
@@ -1,47 +1,47 @@
 @media only screen and (max-width: 600px) {
   .nav-style {
-    font-size: 25px;
-    padding-top: 4%;
-    width: 80%;
-    display: flex;
-    justify-content: space-between;
-    flex-direction: column;
-    align-items: center;
-    position: relative;
```

Comment:
Indenting in this cases allow you to understand the hierarchy of the css classes